cg/services/orders/validation/model_validator/utils.py

```diff
@@ -122,48 +129,54 @@ def get_order_error_details(error_details: list[ErrorDetails]) -> list[ErrorDeta
 
 
 def is_sample_error(error: ErrorDetails) -> bool:
-    return len(error["loc"]) == 3 and error["loc"][0] == "samples"
+    return error["loc"][0] == "samples"
 
 
 def is_case_error(error: ErrorDetails) -> bool:
-    return len(error["loc"]) == 4 and error["loc"][0] == "cases"
+    return "cases" in error["loc"] and "samples" not in error["loc"]
 
 
 def is_case_sample_error(error: ErrorDetails) -> bool:
-    return len(error["loc"]) == 7
+    return "cases" in error["loc"] and "samples" in error["loc"]
 
 
 def is_order_error(error: ErrorDetails) -> bool:
-    return len(error["loc"]) == 1
+    return error["loc"][0] not in ["cases", "samples"]
```

Comment:
will this be more stringent? not sure of how the error will look for an order
```suggestion
    return error["loc"][0] == "order"
```

Reply:
If it is an order error the first entry in "loc" will be the field name. So if the order.name is missing, the loc will just be ("name",) or something.

Reply:
then resolve